distribution/docs/src/main/resources/_contents/config.adoc

```diff
@@ -1,14 +1,15 @@
 Version ${version}. Copyright (c) Codice Foundation
 :imagesdir: target/docs/images
 :toc: left
-:toclevel: 6
-:branding: DDF
+:admin-console: Admin Console
+:command-console: Command Console
 :icons: font
 :example-caption!:
 :source-highlighter: coderay
 :data-uri:
 :sectlinks:
 :sectanchors:
+// :branding: DDF
```

Comment:
No it'll be gone by the last commit. I was just experimenting with maven properties vs. asciidoctor variables.